github-processes.md

```diff
@@ -66,6 +66,16 @@ Here are the rules applied to each branch:
 - On `main` we can be slightly more flexible. It is acceptable to skip QA for PRs that only change typos, documentation or comments. Any changes to source code or tests should be QA approved before merge.
 - In emergency scenarios where an urgent hotfix is required it might be required to skip QA review on `main` branches - this should only happen with QA authorization so QA can retroactively test as soon as possible.
 
+### How to create a release
+
+Creating a release of `if`, `if-plugins` or `if-unofficial-plugins` is achieved by following these steps:
+
+1) core team member creates a tagged release on `main`
```

Comment:
shouldn't we add something about [semver](https://semver.org/)?

Reply:
sure - please suggest some appropriate text here